src/firefly/js/drawingLayers/HiPSMOC.js

```diff
@@ -405,12 +448,31 @@ function updateMocData(dl, plotId) {
                  startIdx, endIdx, updateStatus.storedSidePoints);  // handle max chunk
              updateStatus.processedTiles.push(...moreObjs);
              if (updateStatus.processedTiles.length >= updateStatus.totalTiles) {
-                 abortUpdate(dl, updateStatusAry, plotId, LayerUpdateMethod.byTrueAry);
+                 completeAsyncUpdate(dl, updateStatusAry, plotId, LayerUpdateMethod.byTrueAry);
              }
          }
      }
 }
 
+function getAutoDrawStyle(plot, color) {
+    const [r,g,b]= toRGBA(color);
+    const fov= getFoV(plot);
+    if (!fov) return {style:Style.FILL, color:toRGBAString([r,g,b,1])};
+    if (fov<26) return {style:Style.DESTINATION_OUTLINE, color:toRGBAString([r,g,b,1])};
+
+    let alpha= .7;
```

Comment:
if fov >= 26 and fov <=30, alpha becomes .7 which breaks the transparency increasing when zoomming in behavior. Perhaps this should be .18.

Or L461 can be `if (fov<30)` to prevent this 26-30 gap

Reply:
ok I will fix it.